typed-racket-test/succeed/flonum-typecheck.rkt

```diff
@@ -0,0 +1,123 @@
+#lang typed/racket
+(require racket/flonum
+         typed/rackunit)
+
+
+;;Flonum typechecking tests
+;; -FlZero -Fl
+(check-eq?
```

Comment:
Those tests are primarily aimed at regression testing the typechecks, but they can go if they are superfluous. I couldn't find a good way of "check-expecting" for type errors in the unit tests (check-exn doesn't catch those), hence the block at the end.

It looked to me like this issue concerned fl>= typechecks that shouldn't be passing, which this commit covers.